[2022-01-25 09:30:45,672] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.save_result 2022-01-25T09:25:08.446333+00:00 [queued]>
[2022-01-25 09:30:45,823] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.save_result 2022-01-25T09:25:08.446333+00:00 [queued]>
[2022-01-25 09:30:45,824] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 09:30:45,824] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 09:30:45,825] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 09:30:45,854] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): save_result> on 2022-01-25T09:25:08.446333+00:00
[2022-01-25 09:30:45,880] {standard_task_runner.py:52} INFO - Started process 243 to run task
[2022-01-25 09:30:45,887] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'save_result', '2022-01-25T09:25:08.446333+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpadbyzg2e', '--error-file', '/tmp/tmp11o5jjuv']
[2022-01-25 09:30:45,896] {standard_task_runner.py:77} INFO - Job 16: Subtask save_result
[2022-01-25 09:30:45,982] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.save_result 2022-01-25T09:25:08.446333+00:00 [running]> on host e28fac53771a
[2022-01-25 09:30:46,069] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=save_result
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T09:25:08.446333+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-25T09:25:08.446333+00:00
[2022-01-25 09:30:47,380] {logging_mixin.py:104} INFO - hello
[2022-01-25 09:30:56,986] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 09:30:57,003] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=save_result, execution_date=20220125T092508, start_date=20220125T093045, end_date=20220125T093057
[2022-01-25 09:30:57,051] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 09:30:57,071] {local_task_job.py:146} INFO - Task exited with return code 0
